examples/vault-variable-test/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::Result;
+use spin_sdk::{
+    http::{IntoResponse, Request, Response},
+    http_component,
+    variables,
+};
+
+#[http_component]
+fn handle_vault_variable_test(req: Request) -> Result<impl IntoResponse> {
+    let attempt = std::str::from_utf8(req.body().as_ref()).unwrap();
+    let expected = variables::get("password").expect("could not get variable");
```

Comment:
Nit: I don't think we should be encouraging code that can panic.
```suggestion
    let expected = variables::get("password").context("could not get variable")?;
```